agent-lib/pom.xml

```diff
@@ -24,7 +24,7 @@
   <parent>
     <groupId>org.terracotta</groupId>
     <artifactId>angela-root</artifactId>
-    <version>3.0.6-SNAPSHOT</version>
+    <version>3.0.7-SNAPSHOT</version>
```

Comment:
Shouldn't we bump the version once the PR is merged and a new version is released? Also, I'd wait to release a new version until the concurrent execution problem is fully fixed (i.e. once the ignite changes are made as well)